packages/shared/src/models/IPlaygroundMessage.ts

```diff
@@ -28,6 +28,7 @@ export interface ChatMessage {
 export interface AssistantChat extends ChatMessage {
   role: 'assistant';
   completed: boolean;
+  endTimestamp?: number;
```

Comment:
Instead of having an `endTimestamp` we could simply change the `completed` boolean field to  a number, and having completed undefined when not completed, and the end timestamp value in it when completed.

Thisn would avoid having many fields when completed and simplify the implementation

Reply:
@feloy what do you think about this one ?

Reply:
Yes, ket me change this, and the store below